base/docs/basedocs.jl

```diff
@@ -697,6 +697,13 @@ DevNull
 # doc strings for code in boot.jl and built-ins
 
 """
+    Void
+
+An ordinary type with no fields that is the type of [`nothing`](@ref).
```

Comment:
Just remove ordinary. It may have made more sense in the original context, but here it's unnecessary.